src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Coalesce.txt

```diff
@@ -8,12 +8,24 @@
 >> Coalesce( "", "x" )
 "x"
 
+>> Coalesce("", "2")
+"2"
+
+>> Coalesce("", "a")
+"a"
+
 >> Coalesce( Blank(), "", Blank(), "", "3", "4" )
 "3"
 
+>> Coalesce(Blank(), "", Blank(), "", Blank(), " ", "3", "4")
+" "
+
 >> Coalesce("")
 Blank()
 
```

Comment:
Unrelated to your changes, could you add a case here for an input with only Blanks and empty strings and verify that it returns Blank()?

Reply:
Added